app/views/content_items/briefing.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', title: @content_item.title %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds content-bottom-margin">
+    <div class="responsive-bottom-margin">
+      <%= render "govuk_publishing_components/components/govspeak", {
+        direction: page_text_direction,
+      } do %>
+        <%= raw(@content_item.body) %>
```

Comment:
I think we need to prepend the @content_item.description to the body here - essentially I think the designs want the description text styled as regular <p> text before the body.